ansible/setup-convert2rhel.yml

```diff
@@ -21,10 +19,12 @@
 
   tasks:
 
-    - name: Validate Distribution (CentOS or Oracle Linux) and version
+    - name: Validate source distribution and major version
       fail:
-        msg: "Distribution must be CentOS or Oracle Linux"
-      when: ansible_distribution != 'CentOS' and ansible_distribution != 'OracleLinux'
+        msg: "Distribution must be CentOS/Oracle/Alma/Rocky Linux"
```

Comment:
Should we add that they also should be in major version `7` or `8` because of the condition?

Reply:
Good point. Fixed.